examples/maven-example/run.sh

```diff
@@ -2,7 +2,7 @@
 set -eo errexit
 
 if [ $# -eq 0 ]; then
-  ./mvnw -s ./settings.xml spring-boot:run
+  mvn -s ./settings.xml spring-boot:run
 else
-  ./mvnw -s ./settings.xml spring-boot:run -Dtmc-auth.version="$1"
+  mvn -s ./settings.xml spring-boot:run -Dtmc-auth.version="$1"
```

Comment:
- If you really want to remove the wrapper from the examples, address all of the work outlined in the README.md
- If you are okay with leaving the wrapper in with the example, accept the below suggestion.
```suggestion
  ./mvnw -s ./settings.xml spring-boot:run -Dtmc-auth.version="$1"
```